module/solutions/helper.js

```diff
@@ -1802,8 +1802,7 @@ module.exports = class SolutionsHelper {
 
 				let prefix = CONSTANTS.common.PREFIX_FOR_SOLUTION_LINK
 				let solutionLink = solution?.link
-
-				if (!solutionLink) {
+				if (solutionLink) {
```

Comment:
@MallanagoudaB  it is wrong, if the solutionLink is not there then only we need to create the link